tests/python/gaia-ui-tests/gaiatest/apps/notes/app.py

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+from gaiatest.apps.base import Base
+
+
+class Notes(Base):
+
+    name = 'Notes'
+    manifest_url = "https://marketplace.firefox.com/app/dcdaeefc-26f4-4af6-ad22-82eb93beadcd/manifest.webapp"
+
+    _app_ready_locator = (By.CSS_SELECTOR, 'header')
+    _note_content_locator = (By.ID, 'note-content')
+    _create_new_note_locator = (By.ID, 'button-notebook-add')
+    _save_note = (By.ID, 'button-note-save')
+    _check_note_title_locator = (By.CSS_SELECTOR, '.note .title')
+
+    def launch(self):
+        Base.launch(self)
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(
+                *self._app_ready_locator))))
+
+    def write_note(self, text):
+        Wait(self.marionette).until(expected.element_displayed(
+            Wait(self.marionette).until(expected.element_present(*self._note_content_locator))))
+        note_content = self.marionette.find_element(*self._note_content_locator)
+        note_content.tap()
+        note_content.send_keys(text)
+        self.marionette.find_element(*self._save_note).tap()
+
+    @property
+    def note_title_text(self):
```

Comment:
Per the selector, you can get more than 1 element returned by `find_element`. When that happens, marionette returns the first element that matches. Then, you actually get the first note title. I'd add it in the function name.

On another subject, we can reduce the redundancy: as a title is often a text, there is no need to precise `text` here :smiley: 

Long story short: `first_note_title` sounds like good name. What do you think?